src/main/resources/translations/dhub/strings.properties

```
@@ -210,6 +210,8 @@ command.lookup.player.name=player
 command.lookup.player.description=Lookup a minecraft player.
 command.lookup.user.name=user
 command.lookup.user.description=Lookup a discord user.
+command.lookup.appeal.name=appeal
+command.lookup.appeal.description=Request a review of your Minecraft account status.
```

it's not really the minecraft account; it can also be linked to the discord account
```suggestion
command.lookup.appeal.description=Request a review of your account status.
```